components/Layout/LeadSpace/LayoutLeadSpaceWithCard.vue

```diff
@@ -2,7 +2,7 @@
   <header class="layout-lead-space-with-card">
     <div class="cds--grid">
       <div class="cds--row">
-        <main class="cds--col layout-lead-space-with-card__main">
+        <div class="cds--col layout-lead-space-with-card__div">
```

Comment:
Please rename this class to use something other than `__div`. We can use a class that describes what is in this `<div>`.

Something like:

- `<div class="cds--col layout-lead-space-with-card__hero">`
- `<div class="cds--col layout-lead-space-with-card__content">`